packages/web-react/src/utils/classname.ts

```diff
@@ -6,7 +6,11 @@ export const applyColor =
   <Color>(color: Color) =>
   (className: string): string =>
     `${className}--${color}`;
+export const applySize =
```

Comment:
Hmm, same function with same output with different types. Types should be generic. Function should be one with generic type. Not a matter of this PR. Just for explanation.